Recipely/Recipely/NetworkService/RecipeResponceDTO.swift

```diff
@@ -0,0 +1,30 @@
+// RecipeResponceDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// RecipeResponseDTO
+struct RecipeResponseDTO: Codable {
+    /// Массив с рецептами
+    let hits: [RecipeHitDTO]
+}
+
+/// RecipeHitDTO
```

Comment:
```suggestion
/// RecipeHitDTO
```
дока не соответствует действительности